com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -29,27 +27,16 @@ namespace MLAPI
     public class NetworkManager : MonoBehaviour, INetworkUpdateSystem, IProfilableTransportProvider
     {
 #pragma warning disable IDE1006 // disable naming rule violation check
-        [Browsable(false)]
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-#if UNITY_2020_2_OR_NEWER
         // RuntimeAccessModifiersILPP will make this `public`
-        internal static readonly Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>> __ntable = new Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>>();
+        internal static readonly Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>> __rpc_func_table = new Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>>();
 
-#if UNITY_EDITOR || DEVELOPMENT_BUILD
         // RuntimeAccessModifiersILPP will make this `public`
-        internal static readonly Dictionary<uint, string> __rpc_name_table = new Dictionary<uint, string>();
+        internal static readonly Dictionary<uint, string> __rpc_name_table =
```

Comment:
Not going to lie I think the old way was much cleaner :P using defines like this make code sloppy and hard for people to understand.

Reply:
no worries, I can do that before I merge for sure.

Reply:
done: 3a64122